lib/nodes/Numeric.js

```diff
@@ -14,7 +14,7 @@ const { registerWalker } = require('../walker');
 
 const Node = require('./Node');
 
-const unitRegex = /%|ch|cm|em|ex|in|mm|pc|pt|px|q|rem|vh|vmax|vmin|vw$/i;
+const unitRegex = /%|ch|cm|em|ex|in|mm|ms|pc|pt|px|s|q|rem|vh|vmax|vmin|vw$/i;
```

Comment:
As defined in:

CSS Level 2: https://www.w3.org/TR/CSS2/aural.html#times
CSS Values Level 3: https://drafts.csswg.org/css-values-3/#time
CSS Values Level 4: https://drafts.csswg.org/css-values-4/#time